src/main.js

```diff
@@ -9,3 +8,37 @@ import 'o-layout';
 import './js/demo';
 import './js/main';
 import 'o-autoinit';
+import oTracking from 'o-tracking';
+import CookieMessage from 'o-cookie-message';
+
+const cookieMessage = new CookieMessage();
+const trackingCookieName = cookieMessage.options.consentCookieName;
+
+const hasConsentedToCookies = getCookie(trackingCookieName) === 'true';
```

Comment:
instead of `getCookie` we could use `cookieMessage.shouldShowCookieMessage() === false`

Reply:
Nice!